mt: add page (#7273)
tldr-pages/tldr: pages/linux/mt.md

# mt

> Control magnetic tape drive operation (commonly LTO tape).
> More information: <https://manned.org/mt>.

> - Check tape status:

Reviewer: Perhaps this will read better?
```suggestion
- Check the status of a tape drive:
```

> `mt -f <device> status`

Reviewer 2: We could just put `{{path/to/device}}` everywhere and add ``> Tape devices are usually at `dev/stN` or `dev/nstN`. `` to the description.

> `mt -f {{/dev/[nst|st]N}} status`

Reviewer 3: We don't allow `[]` only this:
```suggestion
`mt -f /dev/{{nst|st}}N status`
```

Author: This is super confusing, who would know what that means? [|] is usually used for an either/or set.
Why not simply leave it at `<device>`, everyone will know what is meant and the **tl;dr** doesn't have to be so cluttered.

Reviewer 3: Let's wait for @sbrl (check this :thread: https://github.com/tldr-pages/tldr/pull/7273#discussion_r737981287)

Reviewer: I suggest picking the most common one and sticking with it. Ref https://www.linuxquestions.org/questions/linux-hardware-18/difference-between-dev-st0-and-dev-nst0-493760/ I suggest `/dev/nst`:
```suggestion
`mt -f {{/dev/nstX}} status`
```
Also, are there any long forms of these arguments at all?

Author: I do not know any long forms. Many tape related commands are actually lacking a lot of really good documentation and tips, that's why I think they're a great addition for this project.
`nst` is indeed the most common one, so let's go with that.

- Rewind the tape to beginning:

`mt -f {{/dev/[nst|st]N}} rewind`

> - Forward `<count>` files, then position tape on first block of next file:
> 
> `mt -f {{/dev/[nst|st]N}} fsf <count>`

Reviewer: ```suggestion
- Move forward a given files, then position the tape on first block of next file:

`mt -f {{/dev/nstX}} fsf {{count}}`
```

> - Rewind the tape, then position tape at beginning of `<count>` file:
> 
> `mt -f {{/dev/[nst|st]N}} asf <count>`

Reviewer: ```suggestion
- Rewind the tape, then position the tape at beginning of the given file:

`mt -f {{/dev/nstX}} asf {{count}}`
```

> - Position tape at the end of valid data:

Reviewer: Perhaps this will read better?
```suggestion
- Position the tape at the end of valid data:
```

`mt -f {{/dev/[nst|st]N}} eod`

- Rewind the tape and unload/eject it:

`mt -f {{/dev/[nst|st]N}} eject`

> - Write EOF (End-of-file) mark at current position:

Reviewer: Perhaps this will read better?
```suggestion
- Write EOF (End-of-file) mark at the current position:
```
